spring-cloud-dataflow-composed-task-runner/pom.xml

```diff
@@ -49,6 +49,10 @@
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc8</artifactId>
```

Comment:
Why not `<scope>runtime</scope>` ?

Reply:
It's consistent with the others. It doesn't really make a difference AFAICT

Reply:
Ok - consistency wins ;)